# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../jbigkit/shared-variant.patch
# Copyright (C) 2004 - 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

I like shared objects ...

  - Rene Rebe <rene@exactcode.de>

--- jbigkit/libjbig/Makefile.vanilla	2008-10-28 16:31:33.000000000 +0100
+++ jbigkit/libjbig/Makefile	2008-10-28 16:35:33.000000000 +0100
@@ -5,9 +5,9 @@
 CC = gcc
 
 # Options for the compiler: A high optimization level is suggested
-CFLAGS = -g -O -Wall -ansi -pedantic # --coverage
+CFLAGS = -g -O -Wall -ansi -pedantic -fPIC # --coverage
 
-all: libjbig.a tstcodec tstcodec85
+all: libjbig.a libjbig.so tstcodec tstcodec85
 
 tstcodec: tstcodec.o jbig.o jbig_ar.o
 	$(CC) $(CFLAGS) -o tstcodec $+
@@ -25,6 +25,12 @@
 	ar rc libjbig85.a jbig85.o jbig_ar.o
 	-ranlib libjbig85.a
 
+libjbig.so: jbig.o jbig_ar.o
+	$(CC) $(CFLAGS) -shared jbig.o jbig_ar.o -o libjbig.so
+
+libjbig85.a: jbig85.o jbig_ar.o
+	$(CC) $(CFLAGS) -shared jbig85.o jbig_ar.o -o libjbig85.so
+
 jbig.o: jbig.c jbig.h jbig_ar.h
 jbig85.o: jbig85.c jbig85.h jbig_ar.h
 jbig_ar.o: jbig_ar.c jbig_ar.h
